Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/security-emailthreatsubmission-review.md

---

> title: "emailThreatSubmission: review"

Reviewer: Title in line 2 and 10, and file name: According to the CSDL, the binding parameter name is threadSubmission, please remove the "email" word at the front.

description: "review threat submission"
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

> # emailThreatSubmission: review

Reviewer: According to the CSDL, the binding parameter if this action is threatSubmission, not emailThreatSubmission. Could you please clarify and fix accordingly?

Reviewer 2: Gen I think you should take care of this

Author: some sentences have been added below. url/file threat submission review are reserved for future usage.

Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Review a threat submission. Only email threats submitted from end users support reviews by an administrator.

Author: ```suggestion
Review a threat submission. Only `emailThreatSubmission` submitted from end user supports `review` action now. `review` action for `urlThreatSubmission` and `fileThreatSubmission` are reserved for future usage.
```

> Review a threat submission. Only `emailThreatSubmission` submitted from end user supports `review` action now. `review` action for `urlThreatSubmission` and `fileThreatSubmission` are reserved for future usage.

Author: ```suggestion
Review a threat submission. Only `emailThreatSubmission` submitted from end user supports `review` action now. `review` actions for `urlThreatSubmission` and `fileThreatSubmission` are reserved for future usage. `urlThreatSubmission` and `fileThreatSubmission` have not been supported for end user now.
```

## Permissions

> One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

Reviewer 2: ```suggestion
One of the following permissions is required to call this API. To learn more permissions, including how to choose one, see [Permissions](/graph/permissions-reference).
```

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|ThreatSubmission.ReadWrite.All|
|Delegated (personal Microsoft account)|N/A|
|Application|ThreatSubmission.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
POST /emailThreats/{emailThreatsId}/review
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|
|Content-Type|application/json. Required.|

## Request body
In the request body, supply a JSON representation of the parameters.

The following table shows the parameters that can be used with this action.

|Parameter|Type|Description|
|:---|:---|:---|

> |category|String|notSpam, junk, phishing, malware. Case insensitive.|

Reviewer 2: ```suggestion
|category|String|The email is being reported as notSpam, junk, phishing, malware. Case insensitive.|
```

---
title: "emailThreatSubmission: review"
description: "Review a threat submission."
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

# emailThreatSubmission: review
Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Review a threat submission. Only [emailThreatSubmission](../resources/security-emailthreatsubmission.md) objects submitted by end users support the review action.

> Review actions for [urlThreatSubmission](../resources/security-urlthreatsubmission.md) and [fileThreatSubmission](../resources/security-filethreatsubmission.md) objects are not supported for end user.

Reviewer 3: By "are not supported for end user", do you mean "Review actions for [urlThreatSubmission](../resources/security-urlthreatsubmission.md) and [fileThreatSubmission](../resources/security-filethreatsubmission.md) objects submitted by end users are not supported"?

Author: Yes. since current there is no urlThreatSubmisson and fileThreatSubmission from end user as our system has not supported it.

|Delegated (personal Microsoft account)|Not supported|
|Application|ThreatSubmission.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
POST /emailThreats/{emailThreatsId}/review
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|
|Content-Type|application/json. Required.|

## Request body
In the request body, supply a JSON representation of the parameters.

The following table shows the parameters that can be used with this action.

|Parameter|Type|Description|
|:---|:---|:---|
|category|String|The email is being reported as notSpam, junk, phishing, malware. Case insensitive.|

## Response

If successful, this action returns a `204 No Content` response code.

## Examples

### Request
The following is an example of a request.
<!-- {
  "blockType": "request",
  "name": "emailthreatsubmissionthis.review"
}
-->
``` http
POST https://graph.microsoft.com/beta/security/threatSubmission/emailThreats/49c5ef5b-1f65-444a-e6b9-08d772ea2059/review
Content-type: application/json

{
  "category": "phishing"
}
```

### Response
The following is an example of the response.

> 
> > **Note:** The response object shown here might be shortened for readability.

Author: ```suggestion
```
